manager/integration/tests/test_rwx.py

```diff
@@ -511,3 +512,60 @@ def test_restore_rwo_volume_to_rwx(set_random_backupstore, client, core_api, vol
     md5sum_pod2 = get_pod_data_md5sum(core_api, pod_name_2, data_path)
 
     assert md5sum == md5sum_pod1 == md5sum_pod2
+
+
+@pytest.mark.skip(reason="TODO")
+def test_rwx_onine_expansion(): # NOQA
+    """
+    Related issue :
+    https://github.com/longhorn/longhorn/issues/2181
+    https://github.com/longhorn/longhorn/issues/2484
+
+    This test case does not cover the UI test mentioned in the related issue's
+    test steps."
+
+    Given
+    - Create a rwx pvc using longhorn storage class of size 1 Gi.
+
+    And
+    - Atach it to a workload (deployment) and write some data.
+
+    When
+    - Expand the volume to 5 Gi
+
+    Then
+    - Expansion of volume is completed.
+    - Data intact
+    - Share manager pod not restart
+    """
```

Comment:
Can we add one more step to write data of size 1.5 Gi, so that we know the expanded data is usable?